server/zio-http-server/src/test/scala/sttp/tapir/server/ziohttp/ZioHttpTestServerInterpreter.scala

```diff
@@ -6,12 +6,12 @@ import sttp.capabilities.zio.ZioStreams
 import sttp.tapir.server.ServerEndpoint
 import sttp.tapir.server.tests.TestServerInterpreter
 import sttp.tapir.tests.Port
-import zhttp.http._
-import zhttp.service.{EventLoopGroup, Server, ServerChannelFactory}
 import zio._
+import zio.http._
+import zio.http.netty.server.NettyDriver
 import zio.interop.catz._
 
-class ZioHttpTestServerInterpreter(eventLoopGroup: EventLoopGroup, channelFactory: ServerChannelFactory)
+class ZioHttpTestServerInterpreter(implicit trace: Trace)
```

Comment:
we'll need to keep the shared Netty resources, as otherwise the tests are *really* slow (as they allocate a thread pool on each run)

Reply:
I had a look at what this would involve. It doesn't seem possible to replace those layers easily due to fact that [`NettyDriver.make`](https://github.com/zio/zio-http/blob/main/zio-http/src/main/scala/zio/http/netty/server/NettyDriver.scala#L67-L82) requires a set of dependencies that are package private to zio-http (such as [`ServerChannelInitializer`](https://github.com/zio/zio-http/blob/main/zio-http/src/main/scala/zio/http/netty/server/ServerChannelInitializer.scala#LL27C31-L27C55)).

Essentially it boils down to copying the entire [NettyDriver](https://github.com/zio/zio-http/tree/main/zio-http/src/main/scala/zio/http/netty/server) package into tapir. Do you think that's feasible and that I should go ahead and do that?

Reply:
Hm ... I guess the option to choose right now is to duplicate the `NettyDriver` - otherwise the tests become really slow (like a couple of minutes). But maybe you could ask on discord / create an issue so that the config is fixed? This used to be possible up until recently.

Reply:
I managed to fix it in ba4e3abf8e6c15270964579286bc4c82d73b4d9a, even if it's somewhat messy.

Reply:
(and the messiness is mostly due to more deconstructed dependencies during the construction of the driver)